index.bs

```diff
@@ -35,6 +35,9 @@ spec:webidl; type:dfn; text:resolve
 .domintro dt code {
     font-size: inherit;
 }
+.domintro::before {
+    content: 'For web developers (non-normative)';
```

Comment:
How about adding:

`text-transform: initial;`

so it isn't forced to uppercase?

Reply:
Good idea, done.